hxat/settings/base.py

```diff
@@ -96,21 +77,11 @@
         # django.db.backends.postgresql_psycopg2 module is deprecated in favor of django.db.backends.postgresq
         # https://docs.djangoproject.com/en/3.0/releases/2.0/#id1
         "ENGINE": "django.db.backends.postgresql",
-        "NAME": os.environ.get(
-            "HXAT_DB_NAME", SECURE_SETTINGS.get("db_default_name", "annotationsx")
-        ),
-        "USER": os.environ.get(
-            "HXAT_DB_USER", SECURE_SETTINGS.get("db_default_user", "annotationsx")
-        ),
-        "PASSWORD": os.environ.get(
-            "HXAT_DB_PASSWORD", SECURE_SETTINGS.get("db_default_password")
-        ),
-        "HOST": os.environ.get(
-            "HXAT_DB_HOST", SECURE_SETTINGS.get("db_default_host", "localhost")
-        ),
-        "PORT": os.environ.get(
-            "HXAT_DB_PORT", SECURE_SETTINGS.get("db_default_port", "5432")
-        ),
+        "NAME": os.environ.get("HXAT_DB_NAME", "annotationsx"),
```

Comment:
I'm running into trouble getting the environment variables from `os.environ.get` when we deploy to our `Dev` environment(`EC2`). A quick fix i found was to add the following to the beginning of the `base.py` file:
```python
from dotenv import load_dotenv, find_dotenv
load_dotenv(find_dotenv())
```
Let me know if that conforms to best practice with `dotenv`

Reply:
we usually set a env var HXAT_DOTENV_PATH=/path/to/dotenv/file in the .bashrc for the user that runs the gunicorn for hxat. This env var is what hxat/asgi.py - hxat/wsgi.py - manage.py look for regarding dotenv.